parts/k8s/manifests/kubernetesmaster-kube-controller-manager-custom.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-controller-manager
+  namespace: kube-system
+  labels:
+    tier: control-plane
+    component: kube-controller-manager
+spec:
+  hostNetwork: true
+  containers:
+    - name: kube-controller-manager
+      image: <img>
+      imagePullPolicy: IfNotPresent
+      command: ["/hyperkube", "controller-manager"]
+      args: [<args>]
```

Comment:
I don't see anything that will properly inject a value into `<args>`

Reply:
src/github.com/Azure/aks-engine/parts/k8s/kubernetesmastercustomdata.yml
line 319
/opt/azure/containers/kubelet.sh will replace the <args>